src/xgov-dapp/src/features/status/TermPoolsTable.tsx

```diff
@@ -7,7 +7,11 @@ interface TermPoolsTableProps {
   govenorData: GovenorTermPoolData[]
   isLoading: boolean
 }
-
+// Indexes of the TermPoll which have been distributed
+const DISTRIBUTED_TERM_POOLS = [0]
```

Comment:
little confused on this, it doesn't look like its ever changed and since it only includes 0 why not just check if the index is zero instead of calling a function that checks the array?

Reply:
Contextually it's a list of indices, it's mostly a habit of excluding things this way with `includes`.

Reply:
i suppose thats better if you anticipate that indices list changing overtime, not gonna take issue with it, still works